app/src/main/res/values/strings.xml

```diff
@@ -21,6 +21,7 @@
     <string name="connect_button_text">Connect</string>
     <string name="choose_server_button_text">Choose server</string>
     <string name="available_servers_title">Available servers</string>
+    <string name="button_cancel_loading">Cancel loading</string>
```

Comment:
Maybe something like "Use a different server" or "Connect to another server" would work better, and at that point maybe even add some more info like the name/address of the server we're currently connecting to.

Reply:
I think showing more details would make this a bit bloated. It also only shows after a second of loading already, so I don't think it should be too verbose.

Reply:
Fixed in 36a7cafae5c02315bc3e047c50c61a7a9b900a5e.